style.css

```diff
@@ -17,28 +94,28 @@ body{
             color:#000000;
             background-color: #b1dfd5;
             width: 100%;
-            height:200px;
+            height:500px;
```

Comment:
Having these heights on the h1-h5 tags seems a bit unusual. What are you trying to accomplish with this?